pipes/WDL/tasks/tasks_assembly.wdl

```diff
@@ -569,7 +569,7 @@ task refine_2x_and_plot {
         grep properly ${sample_name}.all.bam.flagstat.txt | cut -f 1 -d ' ' | tee read_pairs_aligned
         samtools view ${sample_name}.mapped.bam | cut -f10 | tr -d '\n' | wc -c | tee bases_aligned
         #echo $(( $(cat bases_aligned) / $(cat assembly_length) )) | tee mean_coverage
-        python -c "print (float("`cat bases_aligned`")/"`cat assembly_length`") if "`cat assembly_length`">0 else 0" > mean_coverage
+        python -c "print (float("$(cat bases_aligned)")/"$(cat assembly_length)") if "$(cat assembly_length)">0 else 0" > mean_coverage
```

Comment:
In general I like the shift to `$()` notation, however this particular line of code I remember being kind of finicky with all the layers of escaping (python in bash in WDL). Are you sure these values (here and in similar WDL tasks that compute mean coverage) still compute properly? (I feel like we didn't validate any Float outputs like this)

Reply:
Changing to `$()` notation was motivated by trying to suppress the warnings about backtick notation during wdl validation so true issues were easier to spot. I wasn't sure if `$()` woulld cause issues (it shouldn't), but it looks like they **do** compute properly in the CI job:
https://platform.dnanexus.com/projects/F8PQ6380xf5bK0Qk0YPjB17P/monitor/analysis/Fq28Y000xf5x1zQj4gBBvkQ0